dataservice/util/data_gen/data_generator.py

```diff
@@ -0,0 +1,319 @@
+        """
+        Creates sequencing experiments
+        """
+
+        self.e_list = []
+        dt = datetime.now()
+        for i in range(total):
+
+            self.e_data = {
+                'external_id': 'sequencing_experiment_{}'.format(i),
+                'experiment_date': dt - relativedelta.relativedelta(
+                    years=random.randint(1, 3)) + relativedelta.relativedelta(
+                    months=random.randint(1, 6)) +
+                relativedelta.relativedelta(days=random.randint(1, 30)),
+                'experiment_strategy':
+                random.choice
+                (self.experiment_strategy_list),
+                'center': 'Broad Institute',
+                'library_name': 'Test_library_name_{}'.format(i),
+                'library_strand': random.choice(self.library_strand_list),
+                'is_paired_end': random.choice(self.is_paired_end_list),
+                'platform': random.choice(self.platform_list),
+                'instrument_model': random.choice(self.instrument_model_list),
+                'max_insert_size': random.choice([300, 350, 500]),
+                'mean_insert_size': random.randint(300, 500),
+                'mean_depth': random.randint(40, 60),
+                'total_reads': random.randint(400, 1000),
+                'mean_read_length': random.randint(400, 1000)
+            }
+            self.e_list.append(
+                SequencingExperiment(
+                    **self.e_data,
+                    genomic_files=self._create_genomic_files
+                    (random.randint(self.min_gen_files, self.max_gen_files))))
+        return self.e_list
+
+    def _create_genomic_files(self, total):
+        """
+        creates genomic files
+        """
+        self.gf_list = []
+        for i in range(total):
+            self.kwargs = {
+                'file_name': 'file_{}'.format(i),
+                'file_type': 'submitted aligned read',
+                'file_format': random.choice(self.file_format_list),
+                'file_url': 's3://file_{}'.format(i),
+                'md5sum': str(uuid.uuid4()),
+            }
+            self.gf_list.append(GenomicFile(**self.kwargs))
+        return self.gf_list
+
+    def _create_demographics(self, i):
+        """
+        creates demographics
+        """
+        self.data = {
+            'external_id': 'demo_id_{}'.format(i),
+            'race': random.choice(self.race_list),
+            'ethnicity': random.choice(self.ethnicity_list),
+            'gender': random.choice(self.gender_list)
+        }
+        return Demographic(**self.data)
+
+    def _create_diagnoses(self, total):
+        """
+        creates diagnoses
+        """
+        self.diag_list = []
+        for i in range(total):
+
+            self.data = {
+                'external_id': 'diagnosis_{}'.format(i),
+                'diagnosis': random.choice(self.diagnosis_list),
+                'progression_or_recurrence':
```

Comment:
Remove progression_or_recurrence. This field is no longer in the Diagnosis model